.github/workflows/create-release.yml

```
@@ -0,0 +1,47 @@
+# This workflow will install Deno then run Deno lint and test.
+# For more information see: https://github.com/denoland/setup-deno
+
+name: Push to Docs
+
+on:
+  push:
+    tags:
+      - "effection-v*"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    permissions:
+      # Give the default GITHUB_TOKEN write permission to commit and push the
+      # added or changed files to the repository.
+      contents: write
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{github.ref}}
+
+      - name: setup deno
+        uses: denoland/setup-deno@v1
+        with:
+          deno-version: v1.x
+
+      - name: Get Version
+        id: vars
+        run: echo ::set-output name=version::$(echo ${{github.ref_name}} | sed 's/^effection-v//')
+
+      - name: Generate Docs
+        run: deno doc --html --name=effection@$VERSION mod.ts
+        env:
+          VERSION: ${{steps.vars.outputs.version}}
+          
+      - run: tar cfv docs.tar -C docs .
+
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
+        with:
+          files: docs.tar
```

Two suggestions: 

1. let's call them api-docs in case we decide to release other kinds of togs
2. let's compress them so that they can be quickly transferred across the network

```suggestion
      - run: tar cfvz api-docs.tgz -C docs .

      - name: Create Release
        uses: softprops/action-gh-release@v1
        with:
          files: api-docs.tgz
```